src/frontend/components/pages/content/+settings/settings.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="container__page">
+    <div class="list">
+        <div class="list__item" *ngFor="let setting of settings" routerLink="{{setting.route}}">
```

Comment:
Finde ich in diesem Fall übertrieben. Falls wir die Komponente an einem anderen Ort auch noch brauchen, könnte man das dann später immernoch machen (YAGNI). Zuviele Komponenten würde ich vermeiden, macht es nur unübersichtlicher.